docs/libcurl/opts/CURLOPT_CAINFO_BLOB.3

```diff
@@ -37,8 +37,9 @@ Pass a pointer to a curl_blob structure, which contains information (pointer
 and size) about a memory block with binary data of PEM encoded content holding
 one or more certificates to verify the HTTPS server with.
 
-If \fICURLOPT_SSL_VERIFYPEER(3)\fP is zero and you avoid verifying the
-server's certificate, \fICURLOPT_CAINFO_BLOB(3)\fP is not needed.
+If \fICURLOPT_SSL_VERIFYPEER(3)\fP is zero and you avoid verifying the server's
+certificate, any certificates provided by using \fICURLOPT_CAINFO_BLOB(3)\fP
+are not used.
```

Comment:
I think this is confusing. Exactly how would libcurl use the certificate when it doesn't verify it? I think this change adds more questions than it answers!

Reply:
> _If \fICURLOPT_SSL_VERIFYPEER(3)\fP is zero and you avoid verifying the server's certificate_

I think the issue is we have users who are verifying the certificate on their own even though CURLOPT_SSL_VERIFYPEER is zero, so they may misinterpret either version of this. How about _"If \fICURLOPT_SSL_VERIFYPEER(3)\fP is zero then the certificates are not loaded."_

Reply:
That seems a very reasonable text to add!